eventdiscovery-sdk/src/main/res/values-v21/theme_light.xml

```diff
@@ -1,8 +1,11 @@
+<?xml version="1.0" encoding="utf-8"?>
 <resources>
-    <style name="SchedJoules_AppTheme.NoActionBar">
+
+    <style name="SchedJoules_BaseTheme.Light.NoActionBar">
```

Comment:
My motivation for the notes about these `NoActionBar` themes is to possibly take it out from the name as it seems to be the default now and probably forever, to not have the legacy ActionBar, so just have simpler base theme names.
But I understand that extra level is needed to add v21 version. Not sure what would be better naming than, though..
I see also that only 
```
<item name="android:windowDrawsSystemBarBackgrounds">true</item>
<item name="android:statusBarColor">@android:color/transparent</item>
```
are extra for v21. 
So just as an unsure input for what could be in values/styles.xml:
```
<style name="SchedJoules_Theme.Light.Actual"/>
       
    <style name="SchedJoules_Theme.Light" parent="Theme.AppCompat.Light">
        <item name="windowActionBar">false</item>
        <item name="windowNoTitle">true</item>
        <item name="colorPrimary">@color/schedjoules_colorPrimary</item>
        <item name="colorPrimaryDark">@color/schedjoules_colorPrimaryDark</item>
        <item name="colorAccent">@color/schedjoules_colorAccent</item>
```
And then v21 has that 2 extra attributes for `SchedJoules_Theme.Light.Actual`. The name is not very good, I know, so current verision is okay, too, for me.